storey/steps/flatten.py

```diff
@@ -0,0 +1,21 @@
+from itertools import chain
+
+from storey import Flow, Event
+from storey.dtypes import _termination_obj
+
+
+class Flatten(Flow):
+    """
+    Flattens sequences of sequences (i.e lists of lists, sets of sets, etc...) into a single sequence, by default
+    casts sequence to list. When setting to_set=True, casts sequence to set.
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    async def _do(self, event: Event):
```

Comment:
This is an overridden internal method. No need for type annotation here.

Reply:
Is it bad? it's nice for auto-complete

Reply:
I guess it's fine, only inconsistent with the rest of the code.

Reply:
I think adding the annotation to the base class will be better though, no?

Reply:
Just the small issue with `Union[Event, Any]` which is weird as we talked about earlier...